lucene/analysis/nori/src/java/org/apache/lucene/analysis/ko/dict/BinaryDictionary.java

```diff
@@ -150,7 +150,18 @@ protected final InputStream getResource(String suffix) throws IOException {
         throw new IllegalStateException("unknown resource scheme " + resourceScheme);
     }
   }
-  
+
+  public static InputStream getResource(ResourceScheme scheme, String path) throws IOException {
```

Comment:
so .. this basically follows the pattern from JapaneseTokenizer, I think. .. but somehow I don't see where we defined ResourceScheme? We're not referencing the one in kuromoji, right?

Reply:
It's the same as Kuromoji's pattern.

But there is no relation with Kuromoji.
It's already written as an enum in this class.
https://github.com/apache/lucene-solr/blob/d5e44e95175dbf027915e162925057bbcc14200b/lucene/analysis/nori/src/java/org/apache/lucene/analysis/ko/dict/BinaryDictionary.java#L45-L47